.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  extends: ['plugin:vue/essential', '@vue/standard'],
```

Comment:
please change `essential` rule to recommended.
https://eslint.vuejs.org/rules/

Reply:
ok